pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -95,6 +96,13 @@ func (g *wlGroup) IsFinished() bool {
 	return apimeta.IsStatusConditionTrue(g.local.Status.Conditions, kueue.WorkloadFinished)
 }
 
+// IsElasticWorkload returns true if the workload is considered elastic,
+// meaning the ElasticJobsViaWorkloadSlices feature is enabled and the
+// workload has the corresponding annotation set.
+func (g *wlGroup) IsElasticWorkload() bool {
+	return workloadslicing.IsElasticWorkload(g.local)
+}
+
```

Comment:
```suggestion
```

Do we still need? In each place, couldn't we just call `workloadslicing.IsElasticWorkload(g.local)`?

Reply:
I think this helper is consistent with other group functions (e.g., `IsFinished()`), where we don’t need to explicitly unpack group attributes.

That said, I can remove it if it’s a blocker.

Reply:
Im ok either way. I like consistency